pkg/search/controllers_test.go

```diff
@@ -156,7 +192,7 @@ func TestUpdateClusterEventHandler(t *testing.T) {
 			name:       "AddAllEventHandlers_TriggerUpdateClusterEvent_UpdatedClusterAddedToWorkQueue",
 			restConfig: &rest.Config{},
 			client:     fakekarmadaclient.NewSimpleClientset(),
-			restMapper: meta.NewDefaultRESTMapper(nil),
+			restMapper: restmapper.NewDiscoveryRESTMapper(apiGroupResources),
```

Comment:
I have a question: why didn't we set this up before?

Reply:
It didn't cause the test case to fail since it just logging the error:
https://github.com/karmada-io/karmada/blob/079d0ab7a9e59ade7237940625baf15e6fb5c6bd/pkg/search/controller.go#L566